packages/custom-webpack/README.md

```diff
@@ -195,33 +195,20 @@ Builder options:
 Supplemented by the `extra-webpack.config.js` Example:
 
 ```
-const path = require('path');
+const I18nXlfAnnotateAppVersionPlugin = require('./build/i18n-xlf-annotate-app-version.plugin.js');
 
+/**
+ * This is where you define your additional webpack configuration items to be appended to
+ * the end of the webpack config.
+ */
 module.exports = {
-  module: {
-    rules: [
-      {
-        test: /\.less$/,
-        use: [
-          {
-            loader: 'js-to-styles-var-loader',
-          },
-        ],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          {
-            loader: 'js-to-styles-var-loader',
-          },
-        ],
-      },
-    ],
-  },
+  plugins: [
+    new I18nXlfAnnotateAppVersionPlugin()
+  ]
 };
 ```
 
-In this example `module.rules` entry from `extra-webpack.config.js` will be appended to `module.rules` entry from Angular CLI underlying webpack config forcing these loaders to run first, before any of the original loaders defined by the Angular CLI.
+In this example `module.rules` entry from `extra-webpack.config.js` will be appended to `module.rules` entry from Angular CLI's underlying webpack config.
```

Comment:
Please update it with description that is relevant to the example

Reply:
Updated to reflect the custom plugin in the webpack config above